main/utils.js

```diff
@@ -25,7 +25,8 @@ const allowedURLs = [
 ].map(str => new URL(str))
 
 const allowedURLsRegEx = [
-  /https:\/\/beryx.zondax.ch\/v1\/search\/fil\/mainnet\/address\/.*$/
+  /https:\/\/beryx.zondax.ch\/v1\/search\/fil\/mainnet\/address\/.*$/,
+  /https:\/\/docs.filstation.app\/.*$/
```

Comment:
Nitpick: let's add the new line to the top of the list, so that we don't need to change the existing code.

It's unfortunate that https://standardjs.com disallows trailing commas, but that's a different story.

```suggestion
  /https:\/\/docs.filstation.app\/.*$/,
  /https:\/\/beryx.zondax.ch\/v1\/search\/fil\/mainnet\/address\/.*$/
```